src/test/resources/emailconfig.properties

```diff
@@ -0,0 +1,12 @@
+#Gmail Login Credentials
```

Comment:
Does it really need to be Gmail login? Why not yahoo or outlook or some other email provider?